examples/shell/BUILD.gn

```diff
@@ -20,11 +20,12 @@ executable("shell") {
   ]
 
   deps = [
-    "//lib/txt/src",
+    "//dart/runtime:libdart_jit",
```

Comment:
There were some calls to tracing APIs that were trying to add traces via the Dart timeline API. The application wouldn't link otherwise. I will get rid of this.